components/landing/topCollections/TopCollectionsCard/TopCollectionsCard.scss

```diff
@@ -0,0 +1,38 @@
+@import '@/assets/styles/abstracts/variables';
+
+.top-collection-card , .top-collection-card-skeleton {
```

Comment:
can't we replace this file with tailwind classes?

```suggestion
.top-collection-card, 
.top-collection-card-skeleton {
```

Reply:
fixed